frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/modal-footer.tsx

```diff
@@ -82,6 +85,7 @@ export const BlockPoolModalFooter = (props: BlockPoolModalFooterProps) => {
 type BlockPoolModalFooterProps = {
   state: BlockPoolState;
   dispatch: React.Dispatch<BlockPoolAction>;
-  onPoolCreation: (name: string) => void;
   onClick: () => void;
+  action: 'Create' | 'Save' | 'Delete';
+  onPoolCreation?: (name: string) => void;
```

Comment:
this footer is shared by create | delete | update module so no more extra option required